ui/base-tool/src/main/java/org/cru/godtools/base/tool/Activities.kt

```diff
@@ -14,6 +14,8 @@ private const val ACTIVITY_CLASS_TRACT = "org.cru.godtools.tract.activity.TractA
 
 const val EXTRA_SHOW_TIPS = "org.cru.godtools.tract.activity.TractActivity.SHOW_TIPS"
 
+const val SHORTCUT_LAUNCH = "org.cru.goodtools.tract.activity.TractActivity.SHORTCUT_LAUNCH"
```

Comment:
minor nit here, let's make the extra name a bit more generic like `org.cru.godtools.tool.SHORTCUT_LAUNCH`

Reply:
also, you misspelled godtools originally ;)

Reply:
haha that I did! Thanks for catching that 🙂